feed/history/boost_1_75_0.qbk

```diff
@@ -47,11 +47,41 @@ Please keep the list of libraries sorted in lexicographical order.
   * Fixed `named_condition_any` fails to notify ([github interprocess 62])
 ]
 
+* [phrase library..[@/libs/container/ Container]:]
+  * New [classref boost::container::devector devector] container.
+  * Fixed bugs/issues:
+    * [@https://github.com/boostorg/container/issues/152    GitHub #152: ['"Tree-based containers have troubles with move-only types"]].
+    * [@https://github.com/boostorg/container/issues/156    GitHub #156: ['"Compile error with vector"]].
+    * [@https://github.com/boostorg/container/pull/157      GitHub #157: ['"Add missing include"]].
+    * [@https://github.com/boostorg/container/issues/159    GitHub #159: ['"pmr::monotonic_buffer_resource crashes on large single allocations"]].
+    * [@https://github.com/boostorg/container/issues/160    GitHub #160: ['"Usage of uses_allocator needs a remove_cvref_t"]].
+    * [@https://github.com/boostorg/container/issues/162    GitHub #162: ['"small_vector on MSVC x86 call-by-value crash"]].
+    * [@https://github.com/boostorg/container/issues/161    GitHub #161: ['"polymorphic_allocator(memory_resource*) non-standard extension causes headache"]].
+    * [@https://github.com/boostorg/container/pull/163      GitHub #163: ['"container_rebind for small_vector with options"]].
+    * [@https://github.com/boostorg/container/issues/165    GitHub #165: ['"Link error with shared library and memory_resource inline members"]].
+    * [@https://github.com/boostorg/container/pull/166      GitHub #166: ['"Fix encoding error in copyright headers"]].
+    * [@https://github.com/boostorg/container/pull/167      GitHub #167: ['"error: the address of 'msg' will always evaluate as 'true' warning with GCC 4.4"]].
+    * [@https://github.com/boostorg/container/issues/169    GitHub #169: ['"flood of warnings building dlmalloc_ext_2_8_6.c on clang11"]].
```

Comment:
Can these be expressed using `[github_pr container <num>]` for pull requests and `[github container <num>]` for issues?

e.g.
```
* [github container 152] Tree-based containers have troubles with move-only types.
* [github container 156] Compile error with vector.
* [github_pr container 157] Add missing include.
```